### DISCUSSION TITLE
Standardize feature flag names

### DISCUSSION THREAD

**Discussion by efenderbosch** [aws/aws-cdk#12576] [General]
It is easy to understand what the ``aws-cdk:enableDiffNoFail`` feature flag does.

However, feature flag names like ``aws-cdk/secretsmanager:parseOwnedSecretName`` are unclear. I don't know by reading the flag name if if enabling the flag will include or exclude the suffix. A better name for this flag might be ``aws-cdk/secretsmanager:excludeSecretNameSuffix``.

**Comment by indrora** [comment]
I've opened an issue for this, https://github.com/aws/aws-cdk/issues/27058
